input/vocabulary/valueset-signal-filerepo-classification.json

```diff
@@ -0,0 +1,342 @@
+{
+    "resourceType": "ValueSet",
+    "id": "signal-filerepo-classification",
+    "url": "https://signalbhn.org/fhir/us/core/ValueSet/signal-filerepo-classification",
+    "name": "SignalFileRepoClassification",
+    "title": "Signal FileRepo Classification",
+    "status": "draft",
+    "experimental": false,
+    "date": "2023-09-26",
+    "description": "Classification of File Repository",
+    "compose": {
+        "include": [
+            {
+                "system": "https://signalbhn.org/fhir/CodeSystem/signal-filerepo-classification",
+                "concept": [
+                    {
+                        "code": "referral",
+                        "display": "Referral"
+                    },
+                    {
+                        "code": "assessment-request",
+                        "display": "Assessment Request"
+                    },
+                    {
+                        "code": "treatment-record",
+                        "display": "Treatment Record"
+                    },
+                    {
+                        "code": "service-authorizations",
+                        "display": "Service Authorizations"
+                    },
+                    {
+                        "code": "quality-monitoring",
+                        "display": "Quality Monitoring"
+                    },
+                    {
+                        "code": "data-submission",
+                        "display": "Data Submission"
+                    },
+                    {
+                        "code": "invoice",
+                        "display": "Invoice"
+                    },
+                    {
+                        "code": "audit-requested-records",
+                        "display": "Audit Requested Records"
+                    },
+                    {
+                        "code": "credential-requested-records",
+                        "display": "Credential Requested Records"
+                    },
+                    {
+                        "code": "service-organization-feedback",
+                        "display": "Service Organization Feedback"
+                    }
+                ]
+            },
+            {
+                "system": "http://loinc.org"
```

Comment:
Needs to have CONCEPTS for the LOINC system:  See http://hl7.org/fhir/R4/valueset-document-classcodes.html

Reply:
Updated concepts for the LIONC in the valueset-signal-filerepo-classification. Kindly review @enjoysparkling

Reply:
It Looks like the "DocumentReference" profile is yet to be added to the packages. We will bind the value set once it is created.

Reply:
@enjoysparkling I have added the document reference structure definition and bound the valueset. Please review #https://github.com/enjoysparkling/signal-mso-fhir-profiles/pull/104/commits/bc0f1511515147421070c18069d5c7c8fbb5de44